squirrel.api/src/main/java/org/dice_research/squirrel/data/uri/filter/UriFilterConfigurator.java

```diff
@@ -0,0 +1,83 @@
+package org.dice_research.squirrel.data.uri.filter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+
+/**
+ * 
+ * Relational Uri Filter for the AND and OR operators
+ * 
+ * @author Geraldo de Souza Junior (gsjunior@mail.uni-paderborn.de)
+ *
+ */
+
+public class UriFilterConfigurator implements UriFilterComposer {
```

Comment:
This class needs more documentation. Otherwise it is not clear how the `String operator` attribute works.

Reply:
done